src/StackAdmin/Azs.Subscriptions.Admin/readme.md

```diff
@@ -120,7 +120,7 @@ directive:
       verb: Test
       subject: SubscriptionMoveSubscription
     set:
-      subject: MoveSubscription
+      subject: MoveUserSubscription
```

Comment:
can we add an alias in order to minimize the breaking change?

Reply:
ok